js/tests/README.md

```diff
@@ -0,0 +1,17 @@
+## How does Bootstraps's test suite work?
+
+Bootstrap uses [QUnit](http://api.qunitjs.com/), a powerful, easy-to-use JavaScript unit test suite. Each plugin has a file dedicated to its tests in `js/unit/[plugin-name].js`.
+
+* `/test/unit/` contains the unit test files for each Bootstrap plugin.
+* `/test/vendor/` contains third-party testing-related code for the browser environment (jQuery and QUnit)
+* `/test/visual/` contains visual test code which is manually evaluated.
+
+To test usage in a Node.js environment, tests are executed using `npm test`.
+
+
+## How do I add a new test?
+
+1. Locate the file dedicated to the plugin which you need to add tests to.
+2. Add the javascript for your new testcase into the plugin's file within `js/tests/unit/[plugin-name].js`.
```

Comment:
Added. Let me know if this fits what you were thinking though @cvrebert, if not let me know where you were thinking and I'll refactor / rewrite this.